tests/tests.js

```diff
@@ -257,7 +257,8 @@ var runTests = function runTests(arrayFrom) {
 		});
 
 		var hasMap = 'Map' in global;
-		t.test('works with Map objects', { 'skip': !hasMap }, function (te) {
+		var hasMapIterator = hasMap && ((hasSymbols && Map.prototype[Symbol.iterator]) || Map.prototype['@@iterator'] || Map.prototype.iterator);
```

Comment:
I'm not sure about how I change it. Can you show expected code, or edit my commit?